process/inperson-meetings.md

```diff
@@ -0,0 +1,96 @@
+# In-person WebAssembly meetings
+
+In-person meetings are organized typically 1-2 times per year, and are very
+valuable for high-bandwith conversations and informal interactions between
+participants.
+
+## Attending in-person meetings
+
+For in-person meetings, we request that attendees register in advance so that 
+the host may plan capacity accordingly, and so that the meeting notes may
+record who was present. 
+
+The meeting invitation will contain logistics on location, time, address,
+parking, nearby hotels, Wi-Fi, and host contact. This invitation will be sent no
+later than 2 weeks before the meeting, through a pull request on the
```

Comment:
Do you want to keep this strictly NFC? If not, it's probably good to mention that a 3-month lead time is better or even required.

Reply:
I think it makes sense to actually update it too. I did a bit of that already.
Actually this item in particular (advance notification for the location) is mentioned in the hosting part. I don't think it's necessary to have all the logistics available that far in advance though, just basically be committed to the venue and dates.

Reply:
Maybe it makes sense just to remove this "2 weeks" bit entirely. You'll want the venue and dates as far in advance as possible, and at least hotel info more than 2 weeks in advance, but 2 weeks is probably fine for the rest of it.